CMakeLists.txt

```diff
@@ -135,7 +111,7 @@ set(CPACK_PACKAGE_VERSION "${ZKLLVM_VERSION_MAJOR}.${ZKLLVM_VERSION_MINOR}.${ZKL
 if(ZKLLVM_VERSION_INCREMENT)
     string(APPEND CPACK_PACKAGE_VERSION "-${ZKLLVM_VERSION_INCREMENT}")
 endif()
-set(CPACK_PACKAGING_INSTALL_PREFIX "/usr/lib/zkllvm")
+set(CPACK_PACKAGING_INSTALL_PREFIX "usr/lib/zkllvm")
```

Comment:
Please revert, this path is meant to be absolute inside the deb package

Reply:
I have already did it here
https://github.com/NilFoundation/zkLLVM/commit/deba633744f206a50351169e3f42d687a5cf2068